run_brer/run_config.py

```diff
@@ -584,6 +584,11 @@ def run(self, tpr_file=None, **kwargs):
                 assert end_time > start_time
                 if end_time - start_time >= requested_production_time:
                     state_dir = os.path.dirname(self.state_json)
+                    # Previously, when BRER restarted from production --> training,
+                    # the state.json file was overwritten.
+                    # Added changes below maintain previous state.json files from
+                    # completed BRER iterations.
```

Comment:
Previous to what? Let's just describe what the code currently does. How's this?
```suggestion
                    # The state.json file is overwritten below.
                    # We retain previous state.json files from
                    # completed BRER iteration "i" as state_i.json.
```